core/store/orm/orm.go

```diff
@@ -1344,3 +1333,21 @@ func Batch(chunkSize uint, cb func(offset, limit uint) (uint, error)) error {
 		offset += limit
 	}
 }
+
+// SortType defines the different sort orders available.
```

Comment:
Why did this move?

Reply:
It was defined in the middle of the ORM methods. I thought that methods on a type were supposed to be grouped together, so I moved it to after the ORM methods. I'm not sure if this is an "idiomatic Go" recommendation, but I read it somewhere... not super critical.

Reply:
OK :+1: